synapse/config/metrics.py

```diff
@@ -43,8 +43,6 @@ class MetricsConfig(Config):
     def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         self.enable_metrics = config.get("enable_metrics", False)
 
-        self.enable_legacy_metrics = config.get("enable_legacy_metrics", False)
-
```

Comment:
Should we raise a warning telling admins that this setting has been removed and is ignored?

(Probably not? If we're going to rip this out, we should rip it out)

Reply:
yeah, I don't see a big need to. Nobody will be harmed if they leave this in and the option only existed for 4 versions. I wouldn't even be sure that anyone used the options.